CadRevealFbxProvider.Tests/TestSamples/attributes/missing_total_weight_entries.csv

```diff
@@ -0,0 +1,31 @@
+Pole;9.20 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;5
+Pole;9.20 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;6
+Pole;9.20 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;7
+Pole;9.20 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;8
+Pole;9.20 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;9
+Pole;9.20 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;10
+Ladder;18.90 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;11
+Ladder;18.90 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;12
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;13
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;14
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;15
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;16
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;17
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;18
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;19
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;20
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;21
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;22
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;23
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;24
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;25
+Plank;4.54 kg;1;1235678;67;9;175856;;1110;29.02.2020;;;A11;D64LUR-D64ISU;Vedlikehold;Lion Cage;3;1011.00 m³;21;8;8;Yes;Solar Plex;26
```

Comment:
Consdier making hte file shorter. No need for all the data except one line and one Grand total

Reply:
done :)